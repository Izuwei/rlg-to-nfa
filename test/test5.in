A,B
a,b,c
A
A->aA
A->cB
B->bB
