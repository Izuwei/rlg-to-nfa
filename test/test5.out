1,2
abc
1

1,a,1
1,c,2
2,b,2
